.vscode/settings.json

```diff
@@ -27,7 +27,7 @@
     "python.testing.unittestEnabled": false,
     "python.testing.pytestEnabled": true,
     "[github-actions-workflow]": {
-        "editor.defaultFormatter": "redhat.vscode-yaml"
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
why?

Reply:
Because otherwise it's format base on whatever you use for your YAML format, and we need it to be the same for all.